docs/modules/ROOT/pages/hardhat-upgrades.adoc

```diff
@@ -63,10 +66,57 @@ main();
 
 The plugin will take care of comparing `BoxV2` to the previous one to ensure they are compatible for the upgrade, deploy the new `BoxV2` implementation contract (unless there is one already from a previous deployment), and upgrade the existing proxy to the new implementation.
 
+[[beacon-proxies]]
+=== Beacon proxies
+
+You can also use this plugin to deploy an upgradable beacon for your contract with the `deployBeacon` function, then deploy one or more beacon proxies that point to it by using the `deployBeaconProxy` function.
```

Comment:
Not everyone agrees but at this point we've settled on the "upgradeable" spelling.

```suggestion
You can also use this plugin to deploy an upgradeable beacon for your contract with the `deployBeacon` function, then deploy one or more beacon proxies that point to it by using the `deployBeaconProxy` function.
```